stubs/inertia-vue/resources/js/Components/Input.vue

```diff
@@ -1,17 +1,19 @@
-<template>
-    <input class="border-gray-300 focus:border-indigo-300 focus:ring focus:ring-indigo-200 focus:ring-opacity-50 rounded-md shadow-sm" :value="modelValue" @input="$emit('update:modelValue', $event.target.value)" ref="input">
-</template>
+<script setup>
+import { ref } from 'vue';
+
+defineProps(['modelValue']);
+
+defineEmits(['update:modelValue']);
 
-<script>
-export default {
-    props: ['modelValue'],
+const input = ref(null);
 
-    emits: ['update:modelValue'],
+const focus = () => input.focus();
 
-    methods: {
-        focus() {
-            this.$refs.input.focus()
-        }
-    }
-}
+defineExpose({
+    focus,
+});
```

Comment:
I'm not sure what the `focus()` method is actually there for as I can't see it being called from anywhere. I've left it in (with the new syntax) in case it's intended for users, but I'm also happy to remove it.

Reply:
Just for documentation purposes, the focus code seems to be leftover from an attempt to make `<Input autofocus>` work. I've fixed this in d868309 and removed the `defineExpose()` as it wasn't needed.